ci/default.nix

```diff
@@ -30,7 +30,14 @@ let
         inherit (pinned.treefmt-nix) url;
         sha256 = pinned.treefmt-nix.hash;
       };
-      treefmtEval = (import treefmtNixSrc).evalModule pkgs {
+      # Will be upstreamed shortly
+      # error: hash mismatch in file downloaded from 'https://biomejs.dev/schemas/2.1.2/schema.json':
+      patchedTreefmtNixSrc = pkgs.applyPatches {
+        src = treefmtNixSrc;
+        name = "unbreak-ci-20251104";
+        patches = [ ./treefmt-nix-biome-fix.patch ];
```

Comment:
You should avoid vendoring patches wherever possible. In this case, you can fetch the commit's patch from https://github.com/numtide/treefmt-nix/pull/430:

https://github.com/numtide/treefmt-nix/commit/b9710a99ace052fdbfb8e3a636ae8cdc64cf8ec9.patch

See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches

Reply:
That patch did not exist at the time I committed mine :)

It's quite plausible that daniel-fahey chooses to amend to address the comment about not using an eval-time fetcher. I therefore should not fetch the patch because "patches referencing open pull requests will change when the PR is updated and code forges (such as GitHub) usually garbage collect commits that are no longer reachable due to rebases/amends."